src/data/features.ts

```diff
@@ -97,9 +97,9 @@ export const previewFiles = [
 		name: "Files v3",
 		icon: "/ui/icons/folder.png",
 		items: 16,
-		added: "Wednesday, October 6, 2021 42:69 AM",
-		modified: "Thursday, January 1, 1970 00:00",
-		path: "C:\\Users\\Sara\\Desktop\\Files v3"
+		added: "Wednesday, October 6, 2021 9:34 PM",
+		modified: "Thursday, January 1, 1970 00:00 PM",
+		path: "C:\\Users\\Sara\\source\\repos\\Files v3"
```

Comment:
Why the change?

Reply:
VS's default repo folder is %userprofile%\source\repos and obvs files is a vs sln so I thought it would be better to have the project in there rather than Desktop.